movie-app-code-review-main/src/app/components/img/img.ts

```diff
@@ -0,0 +1,26 @@
+import { div } from '@components/tags';
+
+import styles from './img.module.scss';
+
+interface Props {
+  src?: string;
+  alt?: string;
+  className?: string;
+}
+
+export const ImageWithPlaceholder = ({ src = '', alt = '', className = '' }: Props) => {
+  const image = new Image();
+  const wrapper = div(
+    {
+      className: styles.placeholder,
+    },
+    image,
+  );
+  image.src = src as string;
+  image.alt = alt as string;
+  image.className = className as unknown as number as unknown as string;
```

Comment:
This line may cause an error, to avoid this it is better to write: image.className = className as string